app/javascript/components/subject/MoveToHousehold.js

```diff
@@ -1,69 +1,221 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
-import { Form, Row, Col, Button, Modal } from 'react-bootstrap';
+import { Form, Row, Col, Button, Modal, InputGroup, OverlayTrigger, Tooltip } from 'react-bootstrap';
 import axios from 'axios';
+import _ from 'lodash';
 
-import reportError from '../util/ReportError';
+import BadgeHOH from '../util/BadgeHOH';
+import CustomTable from '../layout/CustomTable';
 
 class MoveToHousehold extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      table: {
+        colData: [
+          { field: 'name', label: 'Monitoree', isSortable: true, tooltip: null, filter: this.renderPatientName },
+          { field: 'state_local_id', label: 'State/Local ID', isSortable: true, tooltip: null },
+          { field: 'jurisdiction', label: 'Jurisdiction', isSortable: true, tooltip: null },
+          { field: 'dob', label: 'Date of Birth', isSortable: true, tooltip: null, filter: this.formatDate },
```

Comment:
I think the date is not formatted here because `this.formatDate` is not defined `in MoveToHousehold`. Also, not sure if this is within the scope of this task, but would it also be possible to dynamically move the pagination buttons to another row if the window size isn't that big?

![Screen Shot 2021-01-21 at 11 47 44 AM](https://user-images.githubusercontent.com/9956561/105382834-7f355100-5bde-11eb-8f9e-d6497b561528.png)

Reply:
also, thoughts on making the `Select` buttons medium or small so the rows can be shorter?

Reply:
Good point about the date formatting - I'll add that. As for the responsiveness - yes, that's something I'm working on seeing if I can get in. I actually solved it way back when but there were so many custom table updates it got lost in the rebase so I should be able to get it working again.

Reply:
Fixed dob formatting here: fa73fb4

Reply:
Might be worth seeing if there is an easy fix that wraps the page select below the show dropdown

Reply:
Made ticket for this for next sprint: https://tracker.codev.mitre.org/browse/SARAALERT-1248